oasislmf/gen_test_data/generate_test_data.py

```diff
@@ -0,0 +1,725 @@
+import argparse
+from collections import OrderedDict
+import numpy as np
+import os
+import struct
+
+
+class ModelFile:
+    def __init__(self):
+        pass
+
+    def seed_rng(self):
+        if self.random_seed == 0:
```

Comment:
This condition could be included in the else

Reply:
Sorry, that is a mistake. That value should use current system time as random seed. See suggestion below for change.